### ISSUE TITLE
fail to convert safetensors of XL to diffuser pipeline

### ISSUE THREAD

**Issue by dai-ichiro** [huggingface/diffusers#4562]
#### Reproduction
~~~
python scripts/convert_original_stable_diffusion_to_diffusers.py \
    --checkpoint_path ./sd_xl_base_1.0.safetensors \
    --dump_path ./output_folder \
    --pipeline_class_name StableDiffusionXLPipeline \
    --from_safetensors
~~~

#### Logs
~~~
Traceback (most recent call last):
  File "/mnt/wsl/PHYSICALDRIVE0p1/diffusers-test/diffusers/scripts/convert_original_stable_diffusion_to_diffusers.py", line 170, in <module>
    pipeline_class=pipeline_class,
NameError: name 'pipeline_class' is not defined
~~~
  
I modified line 149 in convert_original_stable_diffusion_to_diffusers.py as follows and everything works.

from
~~~python
class_obj = getattr(library, args.pipeline_class_name)
~~~

to
~~~python
pipeline_class = getattr(library, args.pipeline_class_name)
~~~

**Comment by dai-ichiro** [comment]
related to #4544

**Comment by dai-ichiro** [comment]
fixed in #4590
